styles.css

```
@@ -0,0 +1,369 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap");
+body {
+  font-family: "Poppins", sans-serif;
+  padding: 0rem 1rem;
+}
+.navbar {
+  width: 99%;
+  padding: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  position: relative;
+  background-color: peachpuff;
+  color: black;
+  border-radius: 0.25rem;
+  height: 4rem;
+  text-align: center;
+}
+.navbar .brand {
+  align-items: center;
+  cursor: pointer;
+}
+.navbar .brand-title {
+  font-weight: 700;
+  font-size: x-large;
+  text-align: center;
+}
+section {
+  margin: 9rem;
+  text-align: center;
+  color: green;
+}
+.sec-btn {
+  border: none;
+  color: black;
+  background-color: #ffcba4;
+  padding: 1rem;
+  text-decoration: none;
+  border-radius: 0.5rem;
+  cursor: pointer;
+}
+.footer {
+  background-color: peachpuff;
+  padding: 0.5rem;
+  width: 96%;
+  color: black;
+  font-size: large;
+  border-radius: 0.5rem;
+  text-align: center;
+}
+.footer-content {
+  text-align: center;
+}
+.list-non-bullet {
+  list-style: none;
+  text-align: center;
+}
+
+.display-inline {
+  display: inline;
+}
+.link {
+  box-sizing: border-box;
```

this I think you can give in the top as 
```suggestion
*{
  box-sizing: border-box;
  }
```